packages/react-native/ReactCommon/react/renderer/animations/LayoutAnimationKeyFrameManager.cpp

```diff
@@ -1627,7 +1630,9 @@ Props::Shared LayoutAnimationKeyFrameManager::interpolateProps(
     const PropsParserContext& context,
     Float animationProgress,
     const Props::Shared& props,
-    const Props::Shared& newProps) const {
+    const Props::Shared& newProps,
+    Float viewWidth, 
+    Float viewHeight) const {
```

Comment:
Can you use Size? https://github.com/facebook/react-native/blob/main/packages/react-native/ReactCommon/react/renderer/graphics/Size.h
```suggestion
    const Size& size) const {
```

Reply:
Fixed